ext/testcase.c

```diff
@@ -28,6 +28,14 @@
 #include "ext/repo_helix.h"
 #endif
 
+#ifdef _WIN32
+  #include <direct.h>
+  #define PATH_SEP '\\'
+#else
+  #define PATH_SEP '/'
+#endif
```

Comment:
I thought `/` works as a path separator natively in Win32 as well?

Reply:
Does not seem to be the case (https://ci.appveyor.com/project/wolfv/libsolv/builds/23770809) I'll keep it as it is for now if you don't mind.